guides/doc-Planning_Guide/topics/Glossary.adoc

```diff
@@ -130,7 +130,9 @@ In relationship to a host, erratum is *applicable* if it updates a package insta
 [[varl-Glossary_of_Terms-External_Node_Classifier]]
 *External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
```

Comment:
I still disagree with this. While Puppet introduced it and we follow its format, I still think that the parameters it provides are also very useful in other cases. However, it appears I can't really get through to people so I've given up.

Reply:
I have no context for this comment @ekohl 
I'd gladly help get to where you think this needs to be, but have no idea about Puppet.

Reply:
I've been trying to convince people that Foreman should adopt the term ENC and not consider it a Puppet-only concept. However, this appears to fall on deaf ears. Maybe at this point I'm just ranting.

Reply:
And how would that look? 
```suggestion
*External Node Classifier*:: A construct that provides additional data for a server to use when configuring hosts.
```
Like this?

Reply:
Changed as per comment.